AlpsSDK/RemoteNotificationManager.swift

```diff
@@ -6,18 +6,24 @@
 //  Copyright © 2017 Alps. All rights reserved.
 //
 
-import Foundation
 import UserNotifications
+import UIKit
```

Comment:
I remember why I had to `import UIKit`

Else it doesn't reckon the `UIApplication.shared.registerForRemoteNotifications()` in registerForPushNotifications() function

So I decided to take out Foundation and keep UIKit instead